File: src/lib/ConnectionCollection.java
package lib;

import java.util.*;


public class ConnectionCollection {
  private List<Connection> collection;

  public ConnectionCollection() {
    this.collection = new ArrayList<>();
  }

  public List<Connection> getCollection() {
    return this.collection;
  }

  public void addConnection(Connection connection) {
    this.collection.add(connection);
  }

  public void printConnection() {
    for (Connection connection : collection) {
      connection.print();
    }
  }
}

File: src/backend/lib/connection_collection.py
class ConnectionCollection:
  def __init__(self):
    self.collection = []

  def add(self, connection):
    self.collection.append(connection)
  
  def getCollection(self):
    return self.collection
  
  def print(self):
    for connection in self.collection:
      connection.printConnection()